tests/model_fields/test_imagefield.py

```diff
@@ -181,6 +181,21 @@ def test_pickle(self):
         loaded_p = pickle.loads(dump)
         self.assertEqual(p.mugshot, loaded_p.mugshot)
 
+    def test_image_field_io_closed_file(self):
```

Comment:
Not sure a test is really needed, but if you want, I think an additional assertion in `test_size_method()` after the file is saved might suffice.